OpenRA.Mods.RA/OpenRA.Mods.RA.csproj

```diff
@@ -344,5 +319,7 @@ cd "$(SolutionDir)thirdparty/"
 copy "FuzzyLogicLibrary.dll" "$(SolutionDir)"
 cd "$(SolutionDir)"</PostBuildEvent>
   </PropertyGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Folder Include="Traits\World\" />
+  </ItemGroup>
```

Comment:
That's probably some VS thing. Damn it, I always check the `.csproj` files before commiting...